src/shared-components/common/BrokerDetailsBreadcrumb.tsx

```diff
@@ -1,6 +1,10 @@
 import { Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
```

Comment:
We don't need common folder inside shared-component. Move out Breadcrumb component in shared-components folder.